src/plugins/data/server/saved_objects/search_telemetry.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { SavedObjectsType } from 'kibana/server';
+
+export const searchTelemetry: SavedObjectsType = {
+  name: 'search-telemetry',
+  namespaceType: 'agnostic',
+  hidden: false,
+  mappings: {
+    properties: {
+      errorCount: {
+        type: 'integer',
+      },
+      successCount: {
+        type: 'integer',
+      },
+      averageDuration: {
+        type: 'long',
+      },
+    },
+  },
```

Comment:
It doesn't look like we ever search or aggregate over these fields, can we remove the field mappings altogether? See https://github.com/elastic/kibana/issues/43673

```suggestion
  mappings: {
    dynamic: true,
    properties: {},
  },
```

Reply:
That seems to conflict with the `SavedObjectsType` type...

Reply:
I changed it to `dynamic: false` (which seems to be used elsewhere) and things are working fine now.